vispy/visuals/scrolling_lines.py

```diff
@@ -97,7 +96,7 @@ def __init__(self, n_lines, line_size, dx, color=None, pos_offset=None,
         # set an array giving the x/y origin for each plot
         if pos_offset is None:
             # construct positions as a grid 
-            rows = np.ceil(n_lines / columns)
+            rows = int(np.ceil(n_lines / columns))
```

Comment:
actually this could probably just be `(n_lines - 1) // columns + 1` to avoid float / ceil entirely

Reply:
` int(np.ceil(n_lines / columns))` seems a lot clearer code in intent (in my opinion), and there's effectively zero performance impact, being only called once per `ScrollingLinesVisual`.

I've made the other floor division changes.  I'll do this one as well, if you really want it.

Reply:
I'm fine either way on this one